__tests__/server.test.js

```diff
@@ -90,6 +90,71 @@ describe('Server tests', () => {
     });
   });
 
+  describe('POST /reviews', () => {
+    it('should get the expected status in return for success', () => {
+      const body = {
+        product_id: 12,
+        rating: 4,
+        summary: "I am liking these glasses",
+        body: "They are very dark. But that's good because I'm in erysunny spots",
+        recommend: false,
+        name: "Mister Twister",
+        email: "mister@twister.com",
+        photos: ["google.com"],
+        characteristics: {
+          "42": 4,
+          "41": 2
+        },
+      };
+      return axios.post('http://localhost:3000/reviews', body)
```

Comment:
Watch hard-coded addressing when you migrate to containers/deployment.

Reply:
good call!